app/views/fields/has_one/_form.html.erb

```diff
@@ -18,7 +18,7 @@ The form will be rendered as nested_from to parent relationship.
 
 <%= f.fields_for field.attribute, field.data || field.nested_form.resource.class.new do |has_one_f| %>
   <fieldset class="field-unit--nested">
-    <legend><%= field.nested_form.resource_name.titleize %></legend>
+    <legend><%= t "helpers.label.#{f.object_name}.#{field.nested_form.resource_name}", default: field.nested_form.resource_name.titleize %></legend>
```

Comment:
If there is an existing Rails helper or class method I could use to generate this i18n key, that would be nice, but I looked and couldn't find one.